README.md

```diff
@@ -72,3 +72,8 @@ This section lists the minimum functionality of the Sender and Receiver.
 ## Quality Parameters
 
 Setup the quality parameters of your project (duplication, complexity, coverage, warnings) using the GitHub workflow yml files.
+##  Folder Structure
+![ Folder Structure]( FolderStructure.png)
```

Comment:
Excellent effort at documentation. Explore how you can make this easier for future developers to maintain.